public/js/dashboard.js

```diff
@@ -1,96 +1,91 @@
 "use strict";
 /* global findAncestor */
 
-
 async function sendForm(action, formBody={}) {
-  const response = await fetch(`/user/${action}`, {
-    headers: {
-      "Content-Type": "application/json; charset=utf-8",
-    },
-    mode: "same-origin",
-    method: "POST",
-    body: JSON.stringify(formBody),
-  });
+    const response = await fetch(`/user/${action}`, {
+        headers: {
+            "Content-Type": "application/json; charset=utf-8",
+        },
+        mode: "same-origin",
+        method: "POST",
+        body: JSON.stringify(formBody),
+    });
 
-  if (response.redirected) {
-    window.location = response.url;
-    return;
-  }
-  return await response.json();
+    if (response.redirected) {
+        window.location = response.url;
+        return;
+    }
+    return await response.json();
 }
 
 async function sendCommunicationOption(e) {
-  const { formAction, commOption, csrfToken } = e.target.dataset;
-  sendForm(formAction, { communicationOption: commOption, _csrf: csrfToken })
-    .then(data => {}) /*decide what to do with data */
-    .catch(e => {})/* decide how to handle errors */;
+    const { formAction, commOption, csrfToken } = e.target.dataset;
+    sendForm(formAction, { communicationOption: commOption, _csrf: csrfToken });
```

Comment:
This behavior is slightly different, in that now we don't silently ignore any errors via the `.catch()` stuff. 🤷